plugins/quay/README.md

```diff
@@ -67,3 +81,35 @@ yarn start-backend
 ```
 yarn workspace @janus-idp/backstage-plugin-quay run start
 ```
+
+## For users
+
+### Using Quay plugin in Backstage
+
+Quay is a frontend plugin that enables you to view the information about the container images.
+
+#### Prerequisites
+
+- Your Backstage application is installed and running.
+- You have installed the Quay plugin.
+
+  For installation process, see [Installation](#installation).
```

Comment:
If these two sentences belong to one bullet point, then I recommend putting them on the same line, which you can then even shorten, for example:
```suggestion
- You have installed the Quay plugin. See [Installation](#installation).
```

Reply:
Done